packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -314,24 +313,34 @@ function startServerInNewWindow(
 
   if (process.platform === 'darwin') {
     try {
-      return execa.sync(
+      return await execa(
         'open',
         ['-a', terminal, launchPackagerScript],
         procConfig,
       );
     } catch (error) {
-      return execa.sync('open', [launchPackagerScript], procConfig);
+      return await execa('open', [launchPackagerScript], procConfig);
     }
   }
   if (process.platform === 'linux') {
+    // Awaiting this causes the CLI to hang indefinitely, so this must execute without .sync.
     try {
-      return execa.sync(terminal, ['-e', `sh ${launchPackagerScript}`], {
+      return await execa(terminal, ['-e', 'sh', launchPackagerScript], {
```

Comment:
Why `return await` instead of juts `return`?

Reply:
https://github.com/react-native-community/cli/pull/1230#discussion_r462405982 

Interesting behavior, isn't it?

without await:
<img width="345" alt="image" src="https://user-images.githubusercontent.com/5106466/100374844-2a75bf80-300d-11eb-987c-b1a6b7e834fc.png">

with await:
<img width="341" alt="image" src="https://user-images.githubusercontent.com/5106466/100374870-35305480-300d-11eb-8614-e971c19c9a13.png">